docs/workflows.md

```diff
@@ -0,0 +1,63 @@
+## Workflows
+
+### First Steps
+
+After spinning up the containers following the `README` or `intro.md` you are ready ot rumble. Navigating to the [swagger UI](http://localhost:8000/docs/) provides a good overview of the intended functionalities.
+
+You see that tehre are `generic` endpoints for common actions, wallet specific actions and admin actions. On top of that, you find th trust registry and webhooks being exposed. These are the intended ways of client interactions with these two services.
+
+NOTE: Regardless of the multitude of containers and mechanisms running, [the CloudAPI](http://localhost:8000) and its [SwaggerUI](http://localhost:8000/docs) are the main interaction point intended between clients and the stack. This should be the only endpoint clients should (have to) interact with. There is no need (and no intention to allow that) for client to directly interact with the webhooks or trustregistry container. For a production deployment or a close-to-proudction/smoke-testing deployment, you are well adviced to only expose this endpoint to clients and leave all other endpoints unexposed to the outside world.
+
+#### Trust registry
+
+As a client you can retrieve the trust registry (and NOT alter it). That is intential and all a consumer/client should and is able to do. Altering the trust registry is baked into admin actions and only possible with admin role.
+
+#### Webhooks
+
+A client can subscribe to the webhooks via the cloudapi (as opposed to directly via the webhooks container). This requires only targetting the `/webhooks` endpoint and optionally targetting the `/webhooks/{topic}` subroute by suffixing a topic. Using the auth mechanism (see section below), the app automatically extracts the required info about the wallet (i.e the wallet id and JWT) and retrieves only the associated webhooks with a particular wallet. Failing to authenticate will return a 403 HTTP Error.
+
+### Authentication
+
+Authentication is handled by the CloudAPI and, fom a client perspective, kept simple and convenient. Either, via the Swagger UI auth (padlock butten in UI) or via the HEADER data of your client specifying an `x-api-key`. Regardless of whether ou use the UI or another client, the `x-api-key` value consists of two parts, separated by a dot:
+
+`{role}.{key/password}`
```

Comment:
I'd say key/token instead of password (a password is a self-chosen value, while this is more an api key or token)

Reply:
thanks. 100% agree - token is what goes here but didn't come to mind